src/com/company/set/MyHashSet.java

```diff
@@ -0,0 +1,51 @@
+package com.company.set;
```

Comment:
I'm not sure what are you doing in here but this is not how sets work.
The add and contains method are less efficient than a simple linked list which is not the purpose, please review the lessons and submit a new Code review